lib/ansiblelint/__init__.py

```
@@ -19,5 +19,12 @@
 # THE SOFTWARE.
 """Main ansible-lint package."""
 
+try:
+    import pkg_resources
+
+    __version__ = pkg_resources.get_distribution(__package__).version
+except Exception:
```

Don't use catch-all exceptions.